cmake/modules/FindPython.cmake

```diff
@@ -10,8 +10,8 @@ if(PKG_CONFIG_FOUND)
 endif()
 
 find_program(PYTHON_EXECUTABLE python3 ONLY_CMAKE_FIND_ROOT_PATH)
-find_library(PYTHON_LIBRARY NAMES python3.7 python3.6 python3.5 PATHS ${PC_PYTHON_LIBDIR})
```

Comment:
@Rechi 
So far I know these two variables are not used anywehre. Do you know why they are present here?

Reply:
`PC_PYTHON_LIBDIR` and `PC_PYTHON_INCLUDE_DIRS` is set by `pkg_check_modules(PC_PYTHON python3>=3.5 QUIET)` a few lines above.

Reply:
Okay didn't know that these variables are set from the PKG call.
What would be a lean solution to force the paths from our precompiled binary folders? On my machine somehow the system libraries are preferred.